src/main/scala/org/globalforestwatch/summarystats/forest_change_diagnostic/ForestChangeDiagnosticDataLossYearlyTwoCategory.scala

```diff
@@ -0,0 +1,82 @@
+package org.globalforestwatch.summarystats.forest_change_diagnostic
+
+import frameless.Injection
+import io.circe.syntax._
+import io.circe.parser.decode
+
+
+// Two levels of categorization that lead to yearly forest loss objects
+// (ForestChangeDiagnosticDataLossYearly).
+case class ForestChangeDiagnosticDataLossYearlyTwoCategory(
+                                                         value: Map[String, ForestChangeDiagnosticDataLossYearlyCategory]
+                                                       ) extends ForestChangeDiagnosticDataParser[
+  ForestChangeDiagnosticDataLossYearlyTwoCategory
+] {
+  def merge(
+             other: ForestChangeDiagnosticDataLossYearlyTwoCategory
+           ): ForestChangeDiagnosticDataLossYearlyTwoCategory = {
+
+    ForestChangeDiagnosticDataLossYearlyTwoCategory(value ++ other.value.map {
+      case (key, otherValue) =>
+        key -> value
+          .getOrElse(key, ForestChangeDiagnosticDataLossYearlyCategory.empty)
+          .merge(otherValue)
+    })
+  }
+
+  def toJson: String = {
+    this.value
+      .map {
+        case (key, value) =>
+          key -> value.round
+      }
+      .asJson
+      .noSpaces
+  }
+}
+
+object ForestChangeDiagnosticDataLossYearlyTwoCategory {
+  def empty: ForestChangeDiagnosticDataLossYearlyTwoCategory =
+    ForestChangeDiagnosticDataLossYearlyTwoCategory(Map())
+
+  // There is a breakdown by two levels of categories before the breakdown into
+  // yearly forest loss.
+  def fill(
+            className: String,
```

Comment:
Should this be called "category" instead of "class", based on what we call it in the other classes?

Reply:
Yes, good suggestion, I'll change that.